lib-ee/emqx_ee_bridge/src/emqx_ee_bridge.app.src

```diff
@@ -1,12 +1,13 @@
 {application, emqx_ee_bridge, [
     {description, "EMQX Enterprise data bridges"},
     {vsn, "0.1.9"},
-    {registered, [emqx_ee_bridge_kafka_consumer_sup]},
+    {registered, []},
     {applications, [
         kernel,
         stdlib,
         emqx_ee_connector,
-        telemetry
+        telemetry,
+        emqx_bridge_kafka
```

Comment:
That's the release start order.  However, since there's this kinda of mutual dependency, the `emqx_bridge` app [must wait](https://github.com/emqx/emqx/blob/f01e2f358b6e8011f4d8ffd3ca8f69e2b7cbe326/apps/emqx_bridge/src/emqx_bridge_app.erl#L47) for the EE bridge apps to start.  Otherwise, the bridges might fail to start up because dependencies like `brod` and `wolff` are not running yet.